bin/GameIndex.dbf

```diff
@@ -2585,6 +2591,7 @@ Region = PAL-Unk
 Serial = SCES-54535
 Name   = Everybody's Tennis
 Region = PAL-M11
+eeRoundMode = 0
```

Comment:
ditto and same for all other options without a comment in it, it would take me eons to add a ditto comment for every line. xD